src/common/json.lua

```diff
@@ -49,7 +49,7 @@ local function escape_char(c)
 	return "\\" .. (escape_char_map[c] or string.format("u%04x", c:byte()))
 end
 
-local function encode_nil(val)
+local function encode_nil()
```

Comment:
considering reverting these and making a pr to json.lua repo - maybe we should maintain our own fork though.

Reply:
it's just a copied file - don't think we need to fork